source/fuzz/fuzzer_pass_donate_modules.cpp

```diff
@@ -413,14 +413,24 @@ void FuzzerPassDonateModules::HandleTypeOrValue(
             argument_type_ids));
       }
     } break;
+    case SpvOpSpecConstantOp: {
+      // Make sure the module contains an OpConstant instruction
+      // that has suitable type.
+      new_result_id = FindOrCreateZeroConstant(type_or_value.type_id());
```

Comment:
I have done some reading about OpSpecConstantOp, and actually it is fine for an OpSpecConstantOp to consume id operands that are instances of an OpConstant*, not necessarily OpSpecConstant*.

That means that e.g. if we had:

```
%a = OpSpecConstantOp %t opcode %b %c
```

and if `original_id_to_donated_id` has mapped `%t' to `%t_', `%b' to `%b_' and `%c' to `%c_' already then we could turn the instruction into:

```
%fresh_id = OpSpecConstantOp %t_ opcode %b_ %c_
```

Keeping OpSpecConstantOp would actually be more interesting than using zero.

Would you be able to implement this instead (and ensure that a test covers it)?

Reply:
Fixed in 1ab2f64.